src/config/env.ts

```diff
@@ -1,34 +1,75 @@
-import dotenv from 'dotenv';
-import path from 'path';
-import envVars from './env';
+import dotenv from "dotenv";
+import path from "path";
+import { LogLevelString } from "bunyan";
 
-const filename = process.env.NODE_ENV === 'test' ? '.env.test' : '.env';
-const env = dotenv.config({
-  path: path.resolve(process.cwd(), filename),
-});
+export enum EnvironmentEnum {
+  test = "test",
+  development = "development",
+  production = "production",
+}
 
-function getProxyFromEnvironment(): string{
-  let proxyHost: string = process.env.EXTERNAL_ONLY_PROXY_HOST;
-  let proxyPort: string = process.env.EXTERNAL_ONLY_PROXY_PORT;
+export enum BooleanEnum {
```

Comment:
I wonder why do we need this enum? Can't we just cast/convert our env variable to boolean?

Reply:
yes, but no.  There's a larger change that needs to happen here for how to handle env vars.  The main issue is that the type on `process.env` is a `Record<string, string>` so we can't add booleans to it and we have to be able to use the output to recast into `process.env` in the tests for the time being.

This will change in the future, but we need to leave it as in for now.  The `BooleanEnum` just makes it a bit simpler to not screw up with typos.